src/components/Button.jsx

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+function Button({ count, onClick }) {
+  return (
+    <button type="button" onClick={() => onClick()}>{`Click me(${count})`}</button>
```

Comment:
```suggestion
    <button type="button" onClick={onClick}>
      {`Click me(${count})`}
    </button>
```
이렇게 작성하시는게 가독성이 더 좋은거 같은데 어떠신가요?!

Reply:
네 훨씬 보기가 좋은 거 같습니다!